packages/builder/src/utils/esbuild-index-html.ts

```diff
@@ -0,0 +1,60 @@
+import { ApplicationBuilderOptions } from "@angular-devkit/build-angular";
+import { Spinner } from "@angular/cli/src/utilities/spinner";
+import { env } from "@dotenv-run/core";
+import { DotenvRunOptions } from "@dotenv-run/webpack";
```

Comment:
import type

Reply:
I meant ```import  { type DotenvRunOptions } from "@dotenv-run/webpack";```

I think there are multiple places where I should have done it as well

Reply:
done with @dotenv-run/core